styles/components/header.css

```
@@ -0,0 +1,39 @@
+header {
+  background-color: #262a47;
+  padding: 15px;
+  font-size: large;
+  border-bottom-right-radius: 20px;
+  border-bottom-left-radius: 20px;
+}
```

I would be specific and use a `class` here as well:
```suggestion
.header {
  background-color: #262a47;
  padding: 15px;
  font-size: large;
  border-bottom-right-radius: 20px;
  border-bottom-left-radius: 20px;
}
```